src/derived/evaluateOnce.js

```diff
@@ -0,0 +1,28 @@
+import AnimatedValue from '../core/AnimatedValue';
+import { block, call, always, cond, set } from '../base';
+export function evaluateOnce(node, children = []) {
+  let _children;
```

Comment:
let's not use underscore as prefix for inner variables. It should only be used for object member variables to indicate that the variable should be considered "private"